cmake/onnxruntime_webassembly.cmake

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+file(GLOB_RECURSE onnxruntime_wasm_src CONFIGURE_DEPENDS
+  "${ONNXRUNTIME_ROOT}/wasm/api.h"
+  "${ONNXRUNTIME_ROOT}/wasm/api.cc"
+)
+
+source_group(TREE ${REPO_ROOT} FILES ${onnxruntime_wasm_src})
+
+add_executable(onnxruntime_wasm
+  ${onnxruntime_wasm_src}
+)
+
+add_compile_definitions(
+  ENABLE_ORT_WASM
```

Comment:
should we use the macro 'ENABLE_ORT_WASM' or just use '__wasm__' as predefined by compiler?

take a look at all usage of `ENABLE_ORT_WASM` in the change and it shows like the code are using other compiler predefined macros at the same line. like:
`if defined(__MACH__) || defined(__ANDROID__) || defined(__FreeBSD__) || defined(ENABLE_ORT_WASM)`

Reply:
Good catch. It seems we don't need to use ORT_ macro any more.